machine_learning_hep/analysis/analyzerdhadrons_mult.py

```diff
@@ -567,14 +567,65 @@ def plotternormyields(self):
                                                                     self.v_var_binning))
     def plottervalidation(self):
         if self.p_performval is False:
-            self.logger.fatal("The validation step was set to false. You dont \
+            self.logger.fatal(
+                "The validation step was set to false. You dont \
                                 have produced the histograms you need for the \
                                 validation stage. Please rerun the histomass \
-                                step")
+                                step"
+            )
         self.logger.info("I AM RUNNING THE PLOTTER VALIDATION STEP")
         # You can find all the input files in the self.n_filemass. At the
         # moment we dont do tests for the MC file that would be in any case
         # self.n_filemass_mc. This function will be run on only the single
         # merged LHC16,LHC17, LHC18 file or also on the separate years
         # depending on how you set the option doperperiod in the
         # default_complete.yml database.
+
+        gROOT.SetBatch(True)
+
+        def save_root_object(obj, path, name=None, extension="eps"):
```

Comment:
This is quite generic. can we put it e.g. in `machine_learning_hep.io` or `machine_learning_hep.root`?

Reply:
I couldn't agree more, moved to `root.py`